bin.src/makeLsstCameraRepository.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
Unfortunately, I can't comment on the line I want to, but at line 159 there are hardcoded values for the overscans and prescans.  These are now available in the `segmentation.txt`.

See [here](https://github.com/lsst/obs_lsstSim/blob/syseng3/bin.src/makeLsstCameraRepository.py#L158) for how I dealt with this in the special systems engineering version of phosim.

Reply:
One thing to keep in mind is that the comments in the segmentation.txt files swap the meaning of parallel and serial (and mix up prescan and overscan in what it calls the serial direction).   This file actually produces amp images with 3 pixels serial prescan, 0 pixels "parallel" prescan, and 10 pixels each in serial and parallel overscan.  See, e.g., 
```
/global/projecta/projectdirs/lsst/production/DC2/DC2-R1-2p-WFD-r/output/000094/lsst_a_219976_f2_R43_S22_C17_E000.fits.gz
```

Reply:
OK, then we need to do that translation in the camera generation code.  Has that been filed as an issue?
I agree that the file jibes with Jim's interpretation.  I think the following is what is needed (untested):
```
        oprescan = int(els[15])
        ohoverscan = int(els[16])
        oextended = int(els[17])
        ovoverscan = int(els[18])
        # There is a bug in phosim that makes the long axis the serial direction, but that isn't the
        # correct geometry for the LSST chips.  We will assume the correct orientation, but that means
        # there is no overscan.  We can use the prescan for this.
        # Reorganize for the correct orientation
        prescan = ohoverscan
        hoverscan = ovoverscan
        extended = oprescan
        voverscan = oextended
```

Reply:
Ok - and that's what is committed to this PR now.
I'm hoping Jim can help out with filing an issue (as a DM JIRA?).

Reply:
I think it would be a phosim issue on bitbucket.

Reply:
Is this just a question of changing the description in the segmentation.txt file?  We already addressed the pixel geometry question in https://bitbucket.org/phosim/phosim_release/issues/37/electronic-orientation and I had assumed that we had tacitly agreed not to wrestle with the semantics of parallel vs serial as long as the amplifier file outputs were ok.

Reply:
I guess I'm ok with that.  It just confuses me every time I look at it.  I dislike the cognitive dissonance, but realize the alternative could actually be more work.

Reply:
Are we good here?  Or is there something else to address?

Reply:
We  are good.  I'll approve.